docs/deployment/troubleshooting.md

```diff
@@ -25,6 +25,15 @@ The [OpenFaaS workshop](https://github.com/openfaas/workshop) has more instructi
 
 The Queue Worker component enables asynchronous processing of function requests. The default verbosity level hides the message content, but this can be viewed by setting write_debug to true when deploying.
 
+* Callback request headers
+
+The following additional request headers will be set when invoking the call back url:
+
+| Header            | Description |
+|-------------------|-------------|
+| X-Function-Status | [HTTP status code](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes) returned by the original function call |
```

Comment:
To offset future queries I wonder if the table should cover the full set:
```
X-Call-Id
X-Duration-Seconds
X-Function-Status
X-Request-Id
```

Reply:
What is X-Request-Id?

Reply:
C&P remnant

Reply:
@rgee0 sound like a good suggestion, i've pushed a new commit with those other two headers